nmigen_boards/blackice.py

```diff
@@ -0,0 +1,80 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.lattice_ice40 import *
+
+
+__all__ = ["BlackIcePlatform"]
+
+
+class BlackIcePlatform(LatticeICE40Platform):
+    device = "iCE40HX8K"
+    package = "TQ144:4K"
```

Comment:
This doesn't seem right. I think it should be:
```
device = "iCE40HX4K"
package = "TQ144"
```

Reply:
Ah, I haven't realized the need for the `:4K` suffix. It should not be necessary anymore after m-labs/nmigen@23ed888.

Reply:
Fixed in latest push